dkroutingtool/src/py/main_application.py

```diff
@@ -10,106 +10,75 @@
-    # First retreive the cloud context environment variable
-    try:
-        context = os.environ["CLOUDCONTEXT"]
-    except KeyError as e:
-        raise Exception('!! No Cloud Context Supplied.. are you trying to run local? (--local) !!', e)
-
-    print(f' *   Using Cloud Contex:  {context}')
-    if context.upper() == 'AWS':
-        cloud_client = cloud_context.AWSS3Context(scenario)
-    elif context.upper() == 'GDRIVE':
-        cloud_client = cloud_context.GoogleDriveContext(scenario)
-    else:
-        raise Exception(f"Context Not Implemented: {context}")
-    cloud_client.get_input_data(manual=manual_mapping_mode)
-    return cloud_client
-
-
-def run_routing_from_config(config_file='data/config.json', output_dir=file_config.DEFAULT_OUTPUT_DIR):
-    """Run routing from configuration.
-
-    Args:
-      config_file: Path to config files
-    Returns:
-      Writes outputs to disk.
-    """
-    routing_config = RoutingConfig.from_file(config_file)
-    errors = routing_config.validate()
-    if errors:
-        raise ValueError("Error loading config json:" + '\n'.join(errors))
-
-    print(' *   Building Time/Distance Matrices')
-    # check if node_loader_options are specified
-    config = routing_config.raw_json()
-    if 'node_loader_options' in config.keys():
-        node_data = build_time_dist_matrix.process_nodes(config['node_loader_options'], config['zone_configs'])
-    else:
-        node_data = build_time_dist_matrix.process_nodes()
-    
-    errors = routing_config.validate_against_node_data(node_data)
-    if errors:
-        raise ValueError("Node validation against config failed:" + '\n'.join(errors))
-
-    print(f' *   Starting Model Run at {time.strftime("%H:%M:%S")} (UTC)')
-    # Check if solver options are specified
-    routes_for_mapping_viz, vehicles_viz, zone_route_map = optimization.main(
-        node_data, config, output_dir=output_dir)
-    visualization.main(
-        routes_for_mapping_viz, vehicles_viz, zone_route_map, output_dir=output_dir)
-
-
 def main():
     print(f' *   Model Run Initiated {time.strftime("%H:%M:%S")} (UTC)')
 
-    # Setup output directory.
-    output_dir = file_config.DEFAULT_OUTPUT_DIR
-    file_config.make_output_dir(output_dir=output_dir)
+    # Setup file manager.
+    output_config = OutputPathConfig()
+    # Output to this directory.
+    # TODO: instead output to unique run output directory
+    file_manager = FileManager('.', output_config)
+
+    # Load configuration
+    config_manager = ConfigManager.load(
+        ConfigFileLocations(
+            routing_config_file='data/config.json',
+            build_parameters_file='build_parameters.yml',
+            gps_input_files=GPSInputPaths(
+                gps_file='data/customer_data.xlsx',
+                custom_header_file='data/custom_header.yaml',
+                gps_extra_input_file='data/extra_points.csv'
+            ),
+            manual_edits_input_files=ManualEditsInputPaths(
```

Comment:
@jeshua This assumes the manual files will always be read. I added a quick try/except loop and it ran successfully when the files are absent, so we really just can either check if this is a manual run to disable that loading or check for existence of the files.